platforms/i2c/jhd1313m1_driver.go

```diff
@@ -34,10 +34,18 @@ const (
 	LCD_MOVERIGHT           = 0x04
 	LCD_MOVELEFT            = 0x00
 	LCD_2LINE               = 0x08
+	LCD_CMD                 = 0x80
+	LCD_DATA                = 0x40
```

Comment:
Please use `LCD_SETDDRAMADDR` instead of `LCD_CMD` and `LCD_SETCGRAMADDR` instead of `LCD_DATA`.